src/index.html

```diff
@@ -15,8 +15,107 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
```

Comment:
The `href` attribute in this anchor tag also contains spaces. Please correct it.